torch/_lazy/ts_backend.py

```diff
@@ -1,6 +1,8 @@
+import os
 import torch._C._lazy_ts_backend
 
 
 def init():
     """Initializes the lazy Torchscript backend"""
-    torch._C._lazy_ts_backend._init()
+    if os.environ.get("PARSH_AUTORELOAD_CONTEXT") != "1":
```

Comment:
What is parsh? Why not `TORCH_*` name?

Reply:
@albanD this functionality would be used internally with an improved autoreload library and wouldn't be specific to torch. I'll leave a comment with more context in the internal diff.